src/main/scala/fi/oph/koski/ytr/download/YtrDownloadService.scala

```diff
@@ -53,6 +55,17 @@ class YtrDownloadService(
     )
   }
 
+  def downloadFixtures(): Unit = {
+    val fixtureMonthStart = Some("1980-01")
+    val fixtureMonthEnd = Some("1980-01")
```

Comment:
Loppupäivämäärä on exlusive, eli tämä ei varmaan tällaisenaan lataa vielä mitään dataa YTR-tauluihin.

YtrDownloadSpec:in käyttämät

    val birthmonthStart = "1980-03"
    val birthmonthEnd = "1981-10"

voisi toimia hyvin.